File: src/main/Python/boj_step8/boj1193.py
import sys
cin = sys.stdin.readline

N = int(cin())
sum = 0
for i in range(10_000_000):
    if N >= sum + 1 and N <= sum + i:
        if i % 2 == 0:
            print(f'{i - (sum + i - N)}/{1 + (sum + i - N)}')
            break
        else:
            print(f'{1 + (sum + i - N)}/{i - (sum + i - N)}')
            break
    else: sum += i

File: src/main/Cpp/boj_step8/boj1193.cpp
#include <iostream>
using namespace std;

int main() {
    int N, sum = 0;
    cin >> N;

    for (int i = 0; i < 10000000; i++) {
        if (N >= sum + 1 && N<= sum + i) {
            if (i % 2 == 0) {
                cout << i - (sum + i - N) << '/' << 1 + (sum + i - N);
                break;
            } else {
                cout << 1 + (sum + i - N) << '/' << i - (sum + i - N);
                break;
            }
        } else
            sum += i;
    }
}